activerecord/lib/active_record/railties/databases.rake

```diff
@@ -91,16 +93,17 @@ db_namespace = namespace :db do
 
     # desc 'Runs the "up" for a given migration VERSION.'
     task up: [:environment, :load_config] do
+      raise "VERSION is required" if !ENV["VERSION"].nil? && ENV["VERSION"].empty?
+
       version = ENV["VERSION"] ? ENV["VERSION"].to_i : nil
```

Comment:
This one `version = ENV["VERSION"] ? ENV["VERSION"].to_i : nil` ? Can you explain why it's not required? I am not sure I understand.